### PR TITLE
Remove 0.6 shell workaround code

### PR DESCRIPTION
**Pull request by lindner**

### CODE REVIEW

**Comment by lindner** [comment]
Planning seems to work on my browser...  Did I miss anything?

File: src/planning/plan/planificator.ts [hunk 1]
```diff
@@ -75,9 +68,6 @@ export class Planificator {
     }
     this.consumer = new PlanConsumer(this.arc, this.result);
 
-    this.lastActivatedPlan = null;
-    this.arc.registerInstantiatePlanCallback(this.arcCallback);
```
Comment (mmandlis) [inline src/planning/plan/planificator.ts]: this code seems to no longer be needed in shell, also this is the only use of `un/registerInstantiatePlanCallback` in arc.ts, so it should be ok to remove that too, i think.
thanks for cleaning this up!

Comment (mmandlis) [reply src/planning/plan/planificator.ts]: in web-planner there is a reference to this method 
`this._plansChanged(current, planificator.getLastActivatedPlan())`
the actual plan isn't really used, but worth removing.

Comment (lindner) [reply src/planning/plan/planificator.ts]: super, applying suggested fixes.

**Pull request merged**